subsys/tracing/sysview/tracing_sysview.h

```diff
@@ -639,8 +639,9 @@ void sys_trace_k_thread_info(struct k_thread *thread);
 #define sys_port_trace_pm_device_runtime_enable_enter(dev)		       \
 	SEGGER_SYSVIEW_RecordU32(TID_PM_DEVICE_RUNTIME_ENABLE,		       \
 				 (uint32_t)(uintptr_t)dev)
-#define sys_port_trace_pm_device_runtime_enable_exit(dev) \
-	SEGGER_SYSVIEW_RecordEndCall(TID_PM_DEVICE_RUNTIME_ENABLE)
+#define sys_port_trace_pm_device_runtime_enable_exit(dev, ret)		\
+	SEGGER_SYSVIEW_RecordEndCall(TID_PM_DEVICE_RUNTIME_ENABLE,	\
+				     (uint32_t)ret)
 #define sys_port_trace_pm_device_runtime_disable_enter(dev)		       \
 	SEGGER_SYSVIEW_RecordU32(TID_PM_DEVICE_RUNTIME_DISABLE,		       \
 				 (uint32_t)(uintptr_t)dev)
```

Comment:
nit: for consistency with others, push all `\` to 80cols

Reply:
it is mixing tabs with spaces ...